rust-manifest/src/config.rs

```diff
@@ -9,19 +10,7 @@ pub const DEFAULT_CONFIG_VERSION: &'static str = "1";
 #[derive(Clone, Debug)]
 pub struct Config {
     pub config_version: String,
-    pub remote: Option<ConfigRemote>,
-    pub install: ConfigInstall,
```

Comment:
We don't have to, no. I felt it was appropriate to delete them because they don't do anything now, and I didn't think it would help get the new manifests working.

Can you say more about its purpose?